scheds/rust/scx_chaos/src/lib.rs

```diff
@@ -329,6 +330,7 @@ impl Builder<'_> {
         init_libbpf_logging(None);
 
         let mut open_skel = scx_ops_open!(skel_builder, open_object, chaos)?;
+        open_skel.maps.rodata_data.nr_cpu_ids = *NR_CPU_IDS as u32;
```

Comment:
The theory here was that anything common should be initialised in `scx_p2dq::init_open_skel!`. Could you move this there and remove the duplicate in p2dq source?